src/server_manager/model/digitalocean.ts

```diff
@@ -0,0 +1,41 @@
+// Copyright 2021 The Outline Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {ManagedServer, RegionId} from "./server";
+
+// Keys are cityIds like "nyc".  Values are regions like ["nyc1", "nyc3"].
+export type RegionMap = {
+  [cityId: string]: RegionId[]
+};
+
+export enum Status {
+  ACTIVE,
+  EMAIL_UNVERIFIED,
+  INVALID_BILLING_INFORMATION,
+}
+
+export interface DigitalOceanAccount {
```

Comment:
This can be called Account, since it's already in the digitalocean module

Reply:
Done